server/i18n/test/.gitignore

```diff
@@ -1,2 +0,0 @@
-output.php
```

Comment:
You're right. No I haven't really thought about it and I don't see any practical solution except telling devs to remove this file manually :s